tool/site/client/src/components/prescription-tracker/prescriptionSummaryList.tsx

```diff
@@ -88,7 +88,7 @@ export const PrescriptionSummaryList: React.FC<PrescriptionSummaryProps> = ({
             <StyledPrescriptionActions prescriptionId={id} release />
           }
           {status === "With Dispenser" &&
-            <StyledPrescriptionActions prescriptionId={id} dispense releaseReturn />
+            <StyledPrescriptionActions prescriptionId={id} verify dispense releaseReturn />
```

Comment:
Is there any reason verify is only on this status other than it make the most sense from a business workflow perspective? We could theoretically verify the signature whenever right?